### DISCUSSION TITLE
Orbiting with mouse cursor

### DISCUSSION THREAD

**Discussion by inkoml** [microdee/OpenUnrealSpaceMouse#74] [Ideas]
I love the orbiting mode camera behavior, and wonder if there is an option to trace under mouse cursor in editor instead center view ?

**Comment by microdee** [comment]
hmm not out of the box but it's not a bad idea! I might add that in a 1.2.5 release. Getting the pivot point needs some more finesse anyway. You will have to enable it in config first of course because that can be confusing if you're not expecting it.

**Comment by inkoml** [comment]
ue5 has been released a few days ago. Looking forward to the spaceMouse plugin release. 😍

**Reply by microdee** [reply]
the 5.0 update is already live on the Marketplace, I need to update the prebuilt binaries on Github, I didn't have time yet for the pivot point thing, sorry about that, hopefully I can sit down for that soon
